packages/uswds-core/src/styles/mixins/general/button-disabled.scss

```diff
@@ -7,4 +7,10 @@
   &.usa-button--active {
     @include u-disabled;
   }
+
+  @media (forced-colors: active) {
+    &:not(.usa-button--unstyled) {
+      @include u-disabled-high-contrast-border;
+    }
+  }
```

Comment:
Wouldn't this cause a nested duplicate media query since `@include u-disabled-high-contrast-border;` already has the media query?

https://github.com/uswds/uswds/blob/7413aeafca39fc45565de472a52964f025fbd043/packages/uswds-core/src/styles/mixins/utilities/_disabled.scss#L35-L39

Reply:
Resolved in [a042a7e](https://github.com/uswds/uswds/pull/5295/commits/a042a7e21c0eeb3ae1902578985880508000530e)